src/static/about.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+  <body>
+    <h1>About InVEST</h1>
+    <span id='invest-version'>
+      <h3>InVEST Version: </h3> 
+    </span>
+    <br/><br/>
+    <span id='workbench-version'>
+      <h3>Workbench Version: </h3> 
+    </span>
+    <br/> <br/>
+
+    Documentation: 
+    <a
+      href="http://releases.naturalcapitalproject.org/invest-userguide/latest/"
+      onClick="handleClick(event)">
+      http://releases.naturalcapitalproject.org/invest-userguide/latest/
+    </a>
+    <br/>
+    Homepage: 
+    <a
+      href="https://naturalcapitalproject.stanford.edu/"
+      onClick="handleClick(event)">
+      https://naturalcapitalproject.stanford.edu/
+    </a>
+    <br/>
+    Copyright 2021, The Natural Capital Project
+    <br/>
+    License: 
+    <a
+      href="https://github.com/natcap/invest/blob/master/LICENSE.txt"
+      onClick="handleClick(event)">
+      BSD 3-clause
+    </a>
+    <br/>
+    Project pages: 
+    <br/>
+    <a
+      href="https://github.com/natcap/invest"
+      onClick="handleClick(event)">
+      github.com/natcap/invest
+    </a>
+    <br/>
+    <a
+      href="https://github.com/natcap/invest-workbench"
+      onClick="handleClick(event)">
+      github.com/natcap/invest-workbench
+    </a>
+    <br/>
+
+    <h2>Open-Source Licenses</h2>
+    <table>
+        <tr>
+          <td>PyInstaller</td>
+          <td>GPL</td>
+          <td>
+            <a
+              href='http://pyinstaller.org'
+              onClick="handleClick(event)">
+              http://pyinstaller.org
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>GDAL</td>
+          <td>MIT and others</td>
+          <td>
+            <a
+              href='http://gdal.org'
+              onClick="handleClick(event)">
+              http://gdal.org
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>numpy</td>
+          <td>BSD</td>
+          <td>
+            <a
+              href='http://numpy.org'
+              onClick="handleClick(event)">
+              http://numpy.org
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>pyamg</td>
+          <td>BSD</td>
+          <td>
+            <a
+              href='http://github.com/pyamg/pyamg'
+              onClick="handleClick(event)">
+              http://github.com/pyamg/pyamg
+            </a>
+          </td>
+        </tr>
```

Comment:
Ha, this is ancient!  `pyamg` is no longer shipped with InVEST, so we should pull this out of the other InVEST pane, too.